eriksen.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const _ = require('lodash');
+const ModelProxy = require('./lib/proxy');
+const defaultConfig = {
+  waitTime: 0,
+  primary: null,
+  queue: false,
+  logger: { log: console.log, info: console.log, error: console.error } // eslint-disable-line no-console
```

Comment:
Don't know why I defaulted this to console but it should probably be no-ops by default, like it was before...